http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -687,4 +688,29 @@ class ClusterMonitorActor(
       _ <- metrics.incrementCounter(counterName)
       _ <- metrics.recordResponseTime(timerName, duration)
     } yield ()
+
+  private def findToolImageType(images: Set[ClusterImage]): String = {
+    val terraJupyterImage = "us.gcr.io/broad-dsp-gcr-public/([a-z0-9-_]+):(.*)".r
+    val anvilRStudioImage = "us.gcr.io/anvil-gcr-public/([a-z0-9-_]+):(.*)".r
```

Comment:
Should we get the project names (`broad-dsp-gcr-public`, `anvil-gcr-public`) from some config?